### ISSUE TITLE
crash on use of 'like'

### ISSUE THREAD

**Issue by okennedy** [UBOdin/mimir#354]
```
mimir> select substr(value, 0, 120) from shootings_warm_1 where value like '%Explaining everything for%' limit 80;
Exception in thread "main" java.util.NoSuchElementException: key not found: like
        at scala.collection.MapLike$class.default(MapLike.scala:228)
        at scala.collection.AbstractMap.default(Map.scala:59)
        at scala.collection.MapLike$class.apply(MapLike.scala:141)
        at scala.collection.AbstractMap.apply(Map.scala:59)
        at sparsity.expression.Comparison$.apply(Expression.scala:191)
        at sparsity.parser.Expression$$anonfun$10.apply(Expression.scala:51)
        at sparsity.parser.Expression$$anonfun$10.apply(Expression.scala:49)
        at sparsity.parser.Expression$.comparison(Expression.scala:49)
        at sparsity.parser.Expression$.negation(Expression.scala:36)
        at sparsity.parser.Expression$.conjunction(Expression.scala:28)
        at sparsity.parser.Expression$.disjunction(Expression.scala:21)
        at sparsity.parser.Expression$.expression(Expression.scala:18)
        at sparsity.parser.SQL$.whereClause(SQL.scala:317)
        at sparsity.parser.SQL$.select(SQL.scala:384)
        at sparsity.parser.SQL$.parenthesizedSelect(SQL.scala:375)
        at sparsity.parser.SQL$.statement(SQL.scala:29)
        at mimir.parser.MimirSQL$.basicStatement(MimirSQL.scala:77)
        at mimir.parser.MimirSQL$.statement(MimirSQL.scala:66)
        at mimir.parser.MimirCommand$.command(MimirCommand.scala:31)
        at mimir.parser.LineReaderParser$$anonfun$1.apply(LineReaderParser.scala:81)
        at mimir.parser.LineReaderParser$$anonfun$1.apply(LineReaderParser.scala:81)
        at fastparse.package$.parseInputRaw(package.scala:51)
        at fastparse.package$.parse(package.scala:35)
        at mimir.parser.LineReaderParser.tryParse(LineReaderParser.scala:79)
        at mimir.parser.LineReaderParser.tryReadNext(LineReaderParser.scala:106)
        at mimir.parser.LineReaderParser.hasNext(LineReaderParser.scala:133)
        at mimir.Mimir$.eventLoop(Mimir.scala:140)
        at mimir.Mimir$.interactiveEventLoop(Mimir.scala:121)
        at mimir.Mimir$.main(Mimir.scala:112)
        at mimir.Mimir.main(Mimir.scala)
```

**Comment by okennedy** [comment]
Caused by https://github.com/UBOdin/sparsity/issues/3

**Comment by okennedy** [comment]
Should now be merged in with b8e3a3193a072a6c4622b8611e00402e51862403
